### ISSUE TITLE
Not able to center a custom-checkbox

### ISSUE THREAD

**Issue by Michelle-dev100** [twbs/bootstrap#33302]
Hello,

I want to use a `custom-checkbox` without a label in a table.
The `custom-checkbox` should get **centered in the table-cell** as it does correctly with a normal checkbox, but it is not correctly aligned.  

Please have a look at the following screenshot:

![CustomCheckbox](https://user-images.githubusercontent.com/80244048/110288681-aad29600-7fe8-11eb-878d-9aff40be60e9.png)

I want to provide a live example at JSFiddle to reproduce it easily: [https://jsfiddle.net/3wqovfbd/1/](https://jsfiddle.net/3wqovfbd/1/)

I was not able to find a solution to get it correctly centered.

Thank you very much!

- Operating system and version (Windows, macOS, Android, iOS) - Windows 10 20H2
- Browser and version (Chrome, Firefox, Safari, Microsoft Edge, Opera, Android Browser) - Microsoft Edge 89.0.774.45

**Comment by blakej115** [comment]
I think you are using this in a way that was unintended. They come out much better when you have text inside of the label elements. Regardless, here is a solution: [https://jsfiddle.net/blakej115/gvro5s47/69/](https://jsfiddle.net/blakej115/gvro5s47/69/)

I added the `pl-0` class to your custom-checkbox to get rid of the left-padding.

I created a `.center-label` class that will center the pseudo elements used to create the custom label.

```
.center-label::before,
.center-label::after {
  left: 50%;
  transform: translateX(-50%);
}
```
